calico/network-policy/encrypt-cluster-pod-traffic.mdx

```diff
@@ -39,17 +39,25 @@ This how-to guide uses the following {{prodname}} features:
 - GKE
```

Comment:
Assuming QE testing passes for AKS, I think we can simplify the Before you begin section like this:

**Supported encryption**
- **Inter-node pod traffic, IPv4 only** - traffic leaving a pod from one node destined to a pod on another node
- **Inter-node, host-network traffic, IPv4/IPv6** - traffic generated by the node itself or a host-networked-pod destined to another node or host-networked-pod

**Unsupported**
- Same-node pod traffic (traffic between pods on the same node)
- GKE
- Using your own custom keys to encrypt traffic

**Required**
(as is)

Reply:
I hesitate to say inter-node, host-network traffic is supported globally, as it locks up non-managed clusters, but otherwise this sounds good to me :)

Reply:
I've pushed a change now which simplifies the section a little, but still emphasizes that inter-node, host-network traffic for ipv4/ipv6 is EKS/AKS-only, with either native or Calico CNI